canonical/ubuntu.com
# Capture referrer during a shop purchase

Open this PR again as this [PR](https://github.com/canonical/ubuntu.com/pull/12520) has been closed when the checkout branch was deleted.

## Done

-Capture referrer during a shop purchase

## QA

- Run locally 
- print `flask.session` somewhere e.g. line 77 in `/shop/decorator.py`
- Go to /pro/subscribe?referrer=canonical
- Check `'referrer': 'canonical'` is saved in session

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-1790
